### PR TITLE
Egress Router CNI (OSDOCS-979)

### PR DESCRIPTION
**Pull request by mikemckiernan**
* The ER CNI plug-in is TP for the 4.7 release.

### CODE REVIEW

**Comment by mikemckiernan** [comment]
Draft

/hold

**Comment by mikemckiernan** [comment]
@jboxman , @danielmellado , @weliang1 , PTAL.

If this is technically accurate and as the definition of concision, then I'll take an approval or thumbs up now and add a URL to the content after https://github.com/openshift/openshift-docs/pull/28609 is merged.

**Comment by danielmellado** [comment]
Hi @mikemckiernan , left a few comments also on #28609, sounds good to me! I'll also reply to your JIRA and try to get a slot for a quick meeting on Fri! Thanks!

**Comment by mikemckiernan** [comment]
@openshift/team-documention, PTAL.  After Daniel's review, I rebased and added an xref to the new topic.

/hold cancel

Applies to enterprise-4.7 and--small guess--milestone Future-Release.

Please add the label, milestone, and merge if it looks OK.

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -166,6 +166,12 @@ With the addition of DNS support in the OVN-Kubernetes cluster network provider
 
 For containers interacting with SR-IOV virtual functions (VFs) in Data Plane Development Kit (DPDK) mode, the `app-netutil` library now provides the following functions: `GetCPUInfo()`, `GetHugepages()`, and `GetInterfaces()`. For more information, see xref:../networking/hardware_networks/about-sriov.adoc#nw-sriov-app-netutil_about-sriov[DPDK library for use with container applications].
 
+[id="ocp-4-7-ovn-kubernetes-egress-router-cni"]
+==== Egress router CNI (Technical Preview)
+
+The egress router CNI plug-in is introduced in Technical Preview. The plug-in
```
Comment (adellape) [inline release_notes/ocp-4-7-release-notes.adoc:172]: s/The plug-in enables you to/You can use the plug-in to/

(Per IBMSG)

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -166,6 +166,12 @@ With the addition of DNS support in the OVN-Kubernetes cluster network provider
...
+enables you to deploy an egress router in redirect mode. This egress router provides parity for OVN-Kubernetes compared to OpenShift SDN, but for redirect mode only. The plug-in does not perform in HTTP proxy or DNS proxy modes and this is a difference with the implementation for OpenShift SDN. For more information, see xref:../networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.adoc[Deploying an egress router pod in redirect mode].
```
Comment (adellape) [inline release_notes/ocp-4-7-release-notes.adoc:173]: s/and this/, and this/

**Review by adellape — COMMENTED** [review]
Looks good, couple tiny things. :+1:

**Comment by mikemckiernan** [comment]
@adellape , thank you for the review and corrections.  It's ready for merge when you get a chance.

**Pull request merged**